src/Sylius/Bundle/ApiBundle/spec/CommandHandler/Cart/RemoveItemFromCartHandlerSpec.php

```diff
@@ -27,12 +28,12 @@ final class RemoveItemFromCartHandlerSpec extends ObjectBehavior
     function let(
         OrderItemRepository $orderItemRepository,
         OrderModifierInterface $orderModifier,
-        ProductVariantResolverInterface $variantResolver
+        OrderProcessorInterface $orderProcessor
```

Comment:
If I see correctly, there is no need for this service

Reply:
true